### ISSUE TITLE
Ignore more chars from color contrast rule

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1982]
The following should not fail axe-core's color contrast rule:

```html
<div style="color:#888">☆☆☆☆☆</div>
```

**Comment by straker** [comment]
In 3.5.0 this no longer returns violation and instead returns needs review with message:

```
"Fix any of the following:
  Element content contains only non-text characters"
```

**Comment by padmavemulapati** [comment]
Verified with latest axe-coconut(v4.3.0 (axe-core 3.5.1-canary.d694c95)), working as expected. 
`<div style="color:#888">☆☆☆☆☆</div>` not giving any failure but showing needs review with the message "Element content contains only non-text characters"

![image](https://user-images.githubusercontent.com/47409579/74641813-ec06c680-5197-11ea-8f54-fe100c090025.png)

**Comment by raphaelfabeni** [comment]
I'm using `axe core4.4.1` and I'm still getting this warning.

**Comment by straker** [comment]
@raphaelfabeni Are you able to provide an example of the HTML that is causing the warning? Just to be clear, a warning is the expected behavior for elements that do not contain text (or enough text).

**Comment by raphaelfabeni** [comment]
@straker sure! I didn't know about the warning, so probably that's what I'm getting here.

```html
<h1>Test title<sup>&trade;</sup></h1>
```

![image](https://user-images.githubusercontent.com/1345662/154356793-8501f284-ed40-41c2-b82b-49eaaabdfbad.png)

**Comment by straker** [comment]
Thanks. So the warning is correct since the `<sup>` element does not have "text content" (uses HTML escape character).

**Comment by raphaelfabeni** [comment]
Got it! Thanks for the quick answer @straker ! I also tried to implement a few recommendations I found on the internet (like `aria-label` or `abbr`) but the warning still comes up! I know that is not the topic here, but any suggestions to avoid this warning?

**Comment by straker** [comment]
There's not really a way to avoid the warning since it's just axe informing you it couldn't run color contrast on the element so you'll need to manually verify that it passes. It's marked as Needs Review (the yellow flag icon) and not a violation on the page.

**Comment by raphaelfabeni** [comment]
Got it got it! I appreciate the help here! 🙏 
Thanks @straker

**Comment by ptmkenny** [comment]
I'm getting the same error with `<div class="divider" role="separator">★</div>`, which I'm using like an `<hr>` tag.

Actually, it was very confusing, because when testing with `pa11y` and using the axe engine, the error returned is "Insufficient color contrast", but when testing with the axe Chrome extension, then I see that the actual issue is "Element content contains only non-text characters".

It seems like axe doesn't understand the [separator role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role) , which generally does not contain text content.

**Comment by djalmaaraujo** [comment]
@straker This char: ↗ is reported as a color contrast issue.

**Comment by straker** [comment]
@djalmaaraujo Thanks for letting us know. Would you be able to open a new ticket for it? That way we can track it.
